src/components/Forms/docs/FormComponent.md

```diff
@@ -0,0 +1,24 @@
+Read more @ https://github.com/nbwar/icformable
```

Comment:
We should probably move those docs over as well, at least in part

Reply:
`FormComponent` didn't show up in the docs menu. Not sure if it's because it is an HOC.

Reply:
Oh I mean rather than sending people off to another url for docs, we should have the docs here I think